osbuild/main_cli.py

```diff
@@ -103,6 +103,8 @@ def parse_arguments(sys_argv: List[str]) -> argparse.Namespace:
     # nargs='?' const='*' means `--break` is equivalent to `--break=*`
     parser.add_argument("--break", dest='debug_break', type=str, nargs='?', const='*',
                         help="open debug shell when executing stage. Accepts stage name or id or * (for all)")
+    parser.add_argument("--quiet", "-q", action="store_true",
+                        help="supress normal output")
```

Comment:
`suppress` :)

Reply:
Nice catch! Thanks, fixed, rebased and pushed anew.